### PR TITLE
Add queryable problem types to instructor dashboard

### PR DESCRIPTION
**Pull request by sigberto**
The Queries tab in the instructor dashboard allows them to query by section and problem type. This commit adds
to the types of problems (by block_type) that can be queried.

### CODE REVIEW

**Comment by sigberto** [comment]
@stvstnfrd how does this look?

**Comment by sigberto** [comment]
should work with https://github.com/Stanford-Online/edx-platform/pull/536

File: playbooks/roles/edxapp/defaults/main.yml [hunk 1]
```diff
@@ -702,6 +702,17 @@ EDXAPP_SESSION_SAVE_EVERY_REQUEST: false
 
 EDXAPP_SESSION_COOKIE_SECURE: false
 
+EDXAPP_INSTRUCTOR_QUERY_PROBLEM_TYPES:
+  - 'problem',
```
Comment (stvstnfrd) [inline playbooks/roles/edxapp/defaults/main.yml]: No commas for YAML lists.

File: playbooks/roles/edxapp/defaults/main.yml [hunk 2]
```diff
@@ -702,6 +702,17 @@ EDXAPP_SESSION_SAVE_EVERY_REQUEST: false
...
+  - 'openassessment',
```
Comment (stvstnfrd) [inline playbooks/roles/edxapp/defaults/main.yml]: Where did we get this list from?

This looks like old ORA1, which we are no longer using.

File: playbooks/roles/edxapp/defaults/main.yml [hunk 3]
```diff
@@ -702,6 +702,17 @@ EDXAPP_SESSION_SAVE_EVERY_REQUEST: false
...
+  - 'submit-and-compare',
+  - 'freetextresponse',
+  - 'xblockmufi',
+  - 'qualtricssurvey',
```
Comment (stvstnfrd) [inline playbooks/roles/edxapp/defaults/main.yml]: I didn't know there was anything to submit/query for this XBlock.
CC: @dcadams

File: playbooks/roles/edxapp/defaults/main.yml (hunk 2 above)
Comment (sigberto) [inline playbooks/roles/edxapp/defaults/main.yml]: @stvstnfrd the list of the problem types we want to include came from monica. the specific names come from the way in which the code base distinguishes which xblocks should be queriable, which is a field called "block_type" (https://github.com/Stanford-Online/edx-platform/blob/master/lms/djangoapps/instructor/views/api.py#L1240)

File: playbooks/roles/edxapp/defaults/main.yml (hunk 3 above)
Comment (sigberto) [inline playbooks/roles/edxapp/defaults/main.yml]: @stvstnfrd not sure what you mean?

Comment (stvstnfrd) [inline playbooks/roles/edxapp/defaults/main.yml]: @sigberto I guess my confusion (here and above) is that I suspect we don't want/need to include all of these problem types.

While "qualtricssurvey" is technically a problem, I wasn't aware of it having a grade or any state,
so I'm not sure what exactly instructors would want to (or be able to) query.

We can touch base with @mondiaz for clarification.

Comment (sigberto) [inline playbooks/roles/edxapp/defaults/main.yml]: @stvstnfrd touched base with monica, and we switched qualtrics survey with the survey xblock which is very similar to the poll xblock

File: playbooks/roles/edxapp/defaults/main.yml [hunk 4]
```diff
@@ -702,6 +702,17 @@ EDXAPP_SESSION_SAVE_EVERY_REQUEST: false
 
 EDXAPP_SESSION_COOKIE_SECURE: false
 
+EDXAPP_INSTRUCTOR_QUERY_PROBLEM_TYPES:
+  - 'problem'
+  - 'openassessment'
```
Comment (stvstnfrd) [inline playbooks/roles/edxapp/defaults/main.yml]: Confirm that current ORA2 peer assessment is added, not ORA1.

Comment (stvstnfrd) [inline playbooks/roles/edxapp/defaults/main.yml]: @sigberto Did we confirm that this is from ORA2?

Comment (sigberto) [inline playbooks/roles/edxapp/defaults/main.yml]: @stvstnfrd

**Comment by sigberto** [comment]
@stvstnfrd squashed

**Pull request merged**